pkg/manifests/manifests.go

```diff
@@ -1657,13 +1677,12 @@ func (f *Factory) PrometheusK8s(grpcTLS *v1.Secret, trustedCABundleCM *v1.Config
 					Replacement:  "alerts",
 				},
 			},
+			MetadataConfig: &monv1.MetadataConfig{
```

Comment:
This change seems to not be related, was it intentional?

Reply:
Yes but it should be commented. Basically we don't want/need to forward metadata information to the telemeter server.

It comes from https://github.com/openshift/cluster-monitoring-operator/pull/1416 which was the previous PR that exercised the receive path of telemeter.

Reply:
Yes, Telemeter forwards rw requests to Thanos, and Thanos doesn't have metadata ingestion yet. We can still send it, but it would just be dropped. 🙂

Reply:
Did you still want to add a comment for this @simonpasquier ?